hoot-core/src/main/cpp/hoot/core/visitors/geometrymodifiers/WayToIntersectionGeoModifier.h

```diff
@@ -51,20 +51,21 @@ class WayToIntersectionGeoModifier : public GeometryModifierAction
   struct IntersectionInfo
   {
     CoordinateExt intersectionPoint;
-    long segNodeId1, segNodeId2;
+    long segNodeId1;
+    long segNodeId2;
   };
 
   static std::string className() { return "hoot::WayToIntersectionGeoModifier"; }
 
-  QString getCommandName() const { return "way_to_intersection"; }
-  QList<QString> getParameterNames() const { return QList<QString> {}; }
+  virtual QString getCommandName() const override { return "way_to_intersection"; }
+  virtual QList<QString> getParameterNames() const override { return QList<QString>(); }
 
-  void parseArguments( const QHash<QString, QString>& arguments ) { (void)arguments; /* unused */ }
-  bool process( const ElementPtr& pElement, OsmMap* pMap );
+  virtual void parseArguments(const QHash<QString, QString>& /*arguments*/) override { }
```

Comment:
Commenting out the parameter name removes the compiler warning/error for unused parameter so that the function can just be left empty.